src/ol_infrastructure/applications/mitopen/Pulumi.applications.mitopen.Production.yaml

```diff
@@ -3,6 +3,48 @@ secretsprovider: awskms://alias/infrastructure-secrets-production
 encryptedkey: AQICAHiiGjYUolrtj8PCnScLM7oLAdMl8nJrLjQjnqyl1LykYgGq710cJf4taf7FUfw+iIGNAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM2JcEQu/tozu3tO8TAgEQgDvnl65jF/6aC66eb97jIrt3/CIL6a5XcVwv9E8EWgWlrX3cJadv9f2X0bcVLKiGPImjG4qTL6m5sDN7Ig==
 config:
   consul:address: https://consul-apps-production.odl.mit.edu
+  heroku:app_id:
+    secure: v1:mdVwWs7IkyH7o5j7:49eMfOoVshilsOCouixsxdQSGehSeqRHF3DhPWp2UIiE1clEoyoKqOodkpBQ27rik/MIDg==
+  heroku:user: "odl-devops"
+  heroku_app:interpolation_vars:
+    cors_urls:
+    - "https://ocw-preview.odl.mit.edu"
+    - "https://draft.ocw.mit.edu"
+    - "https://www.ocw.mit.edu"
+    - "https://ocw.mit.edu"
+    - "https://live.ocw.mit.edu"
+    etl_micromasters_host: "micromasters.mit.edu"
+    etl_xpro_host: "xpro.mit.edu"
+    mailgun_sender_domain: "discussions-mail.odl.mit.edu"
+    sso_url: "sso.ol.mit.edu"
+  heroku_app:vars:
+    CELERY_WORKER_MAX_MEMORY_PER_CHILD: 250000
+    DEBUG: false
```

Comment:
```suggestion
    DEBUG: "false"
```

Reply:
I thought I copied this from the most recent version. :-| 👍